src/sentry/templatetags/sentry_react.py

```diff
@@ -1,13 +0,0 @@
-from django import template
-
-from sentry.utils import json
-from sentry.web.client_config import get_client_config
-
-register = template.Library()
-
-
-@register.simple_tag(takes_context=True)
-def get_react_config(context):
```

Comment:
Removing this will require a few pull requests as getsentry also uses this template tag in the admin and sentry/layout templates. You could add the context data first and update the templates here, then update getsentry, and then remove the template tag though.

Reply:
Good point, I'll bring the file back